### DISCUSSION TITLE
7412 dauert zu lange zum Verbinden?

### DISCUSSION THREAD

**Discussion by hurzl** [Freetz-NG/freetz-ng#914] [push firmware] [answered]
Nach `push_firmware` und Stecker-ziehen dauert es ca. 30 Sekunden `Waiting .........` , bis der `ping 192.168.178.1` erfolgreich wird. Danach funktioniert `ftp` natürlich nicht mehr. Habe feste IP 192.168.178.2 am pc (Manjaro) und direkte Kabel-Verbindung. `Networkmanager.service` gestoppt. `ping` zeigt ca. 30s `Zielhost nicht erreichbar`.
Auch manuelles Löschen und Neusetzen der IP direkt nach dem Wiedereinstecken hilft nicht.
Firmware ist die aktuelle 6.88 von avm.

```
 * No reply from box, assuming switch-off or restart. Trying to re-detect box.
   Waiting ....................................................................... found!

ftp: connect to address 192.168.178.1: Verbindungsaufbau abgelehnt
ftp: no response from host
 * Detected linux_fs_start: '<unset>'
 * Designated linux_fs_start: 1

 * No reply from box, assuming switch-off or restart. Trying to re-detect box.
   Waiting . found!

 * Box is back up again, initiating transfer.

ftp: connect to address 192.168.178.1: Verbindungsaufbau abgelehnt
ftp: no response from host
Not connected.
Debugging on (debug=1).
Not connected.
Not connected.
Not connected.
Not connected.
Not connected.
Not connected.

done
```

**Comment by MaxineMuster** [comment]
Wenn es 30 Sekunden dauert vermute ich mal, dann ist die Box vielleicht  schon wieder hochgefahren?
Der Timeslot für den FTP ist halt nur wenige Sekunden.

Du könntest versuchen, in einem anderen Fenster ständig die IP zu setzen  (natürlich mit root-Rechten)
 `while true ; do ifconfig <XY> 192.168.178.3 up ; sleep 1; done`

Oder du schaltest einen Switch dazwischen, damit der Link am PC immer aktiv ist.

Ich musste bei der 7412 übrigens nach dem ersten (erfolgreichen) Finden (also beim zweiten  Waiting ....) 
```
* No reply from box, assuming switch-off or restart. Trying to re-detect box.
   Waiting .......... found!

 * Detected linux_fs_start: '1'
 * Designated linux_fs_start: 0

 * No reply from box, assuming switch-off or restart. Trying to re-detect box.
   Waiting .............. found!

 * Box is back up again, initiating transfer.

```

die Box nochmal stromlos machen, der automatische Reboot hat irgendwie nicht geklappt.

**Comment by hurzl** [comment]
Danke,  auf die Idee mit dem Switch kam ich noch nicht, hat aber auch nicht geholfen, ist das gleiche Ergebnis. Wiederholtes `ip addr rep ...` bringt leider auch nichts. Der Ping kommt mind. 30s lang nicht durch, das ist dann zu spät. Die LED am Switch bleibt auch so lange  aus.

**Reply by Conan179** [reply]
Was mir als info fehlt, wie ist die ip addresse von deinem Adapter eingestellt?
Kannst du mal ein Live Linux testen?

**Reply by hurzl** [reply]
s.o. "Habe feste IP 192.168.178.2 am pc (Manjaro) und direkte Kabel-Verbindung" Ein live-arch habe ich auch schon probiert.

**Reply by Conan179** [reply]
Ein Screenshot wäre mal was, weil, nicht ein ping geht nach 30sec durch, sondern nach 30sec lauter pings kommt eine ping antwort, "ip addr rep ..." ist ein merkwürdiger befehl.

**Reply by hurzl** [reply]
das ist die heutige Version von `ifconfig`. Leider.

**Comment by hurzl** [comment]
Nachdem ich die Box vorgestern irgendwie falsch geflasht habe (da ging das noch, sie war nämlich auf 6.86 o.ä.!), startete sie nicht mehr. Dann habe ich sie mit der AVM-Recovery.exe auf die neueste Version gebracht. Läuft auch einwandfrei. Nur flashen kann ich sie nun nicht mehr. Gibt es also vielleicht irgendwelche Neuerungen, die das verhindern sollen?

**Reply by MaxineMuster** [reply]
Wüsste ich nicht. Aber wenn du Zugriff auf einen Windows-PC hast, wäre das mit dem Recovery der "Notvorschlag" gewesen. 
Allerdings ist die Beobachtung, dass beim Boot der Switch keinen Link anzeigt etwas komisch. Denn auch das Recovery funktioniert so, dass es direkt nach dem Booten was übers Netz zur Box schickt.
Eine Möglichkeit, die Box "sicher" in den FTP-Modus zu bekommen, war (zumindest früher bei älteren Boxen) immer, ein Recovery einer anderen Box zu starten, was dann beim Testen der Box fehlschlägt die Box aber im FTP lässt.

Nur wenn zu Beginn kein Link da ist, dürfte das auch nicht klappen.

Allerletzte Möglichkeit wäre dann nur noch der Hardware-Eingriff, um über die serielle Konsole die Box im FTP zu halten und auch zu sehen, was so passiert...

Wenn Recovery generell läuft, könntest du auch ein älteres Recovery versuchen, hab in der waybackmachine mal eines rausgesucht:
https://web.archive.org/web/20181013135844if_/http://download.avm.de/fritzbox/fritzbox-7412/deutschland/recover/FRITZ.Box_7412.06.85.recover-image.exe

Du kannst dich da auch "durchsuchen", ob du eine andere Version willst/findest:
https://web.archive.org/web/20240000000000*/https://download.avm.de/fritzbox/fritzbox-7412/deutschland/recover/

**Comment by hurzl** [comment, marked as answer]
Da auch die AVM-Recovery unter Windows die Box nicht mehr gefunden hat ("gescheitert(1460)"), habe ich nun per Web-Interface das image neu installiert und jetzt funktioniert der ftp-server wieder beim Einschalten. :-/
